app/assets/stylesheets/application.scss

```diff
@@ -11,3 +11,11 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+
+body {
+font-family: Circular, -apple-system, BlinkMacSystemFont, Roboto, Helvetica Neue, sans-serif;
```

Comment:
Since some of these fonts are system-specific, I would adjust those [e.g., -apple-system would not be available for windows users and BlinkMacSystemFont (possibly Circular as well) are not commonly available unless @ imported).